pkg/commands/build/build.go

```diff
@@ -63,15 +55,14 @@ func NewCmdBuild(
 	rf *resmap.Factory,
```

Comment:
looks like `fs` is no longer needed here, so it should prob just be removed

Reply:
`fs` is needed in `RunBuild` in line 78.